bin/compile-extension-mongodb

```diff
@@ -5,14 +5,12 @@ source $(dirname $0)/compile-extensions-common
 
 travis_time_start
 
-if [[ $VERSION =~ ^7 || $VERSION =~ ^master$ ]]; then
+if [[ $VERSION =~ ^master$ || $VERSION =~ snapshot$ ]]; then
 	git clone https://github.com/mongodb/mongo-php-driver.git
 	pushd mongo-php-driver
 	git submodule update --init
 	phpize
-	./configure
-	make  all
-	make install
+	./configure && make && make install
```

Comment:
This is not necessary, as we set `errexit`.

Reply:
Otherwise it looks good.

Reply:
Amended the commit to remove this change.